src/pages/Models/Details/generators.js

```diff
@@ -221,7 +232,10 @@ export function generateMachineRows(modelStatusData, filterByApp) {
         {
           content: (
             <>
-              <div>{machineId}</div>
+              <div>
```

Comment:
This div negates the need for the fragment wrapper or vice versa.

Reply:
The `<a>` is also part of the element so we still need the fragment.